scripts/riskmetric.R

```diff
@@ -0,0 +1,47 @@
+library(glue)
+library(dplyr)
+library(riskmetric)
+
+riskmetric_score <- "tidyCDiSC" %>%
+  pkg_ref() %>%
+  as_tibble() %>%
+  pkg_assess() %>%
+  pkg_score() %>%
+  pull("pkg_score")
```

Comment:
Hi @Jeff-Thompson12, here are the details of running this code locally:

![image](https://user-images.githubusercontent.com/17878953/191780771-66f1e1d4-d70a-4ee5-9a40-583885e4614c.png)